next.config.mjs

```diff
@@ -66,7 +66,7 @@ const nextConfig = {
     NIMI_API_BASE_URL: process.env.NIMI_API_BASE_URL,
     NIMI_API_DEV_BASE_URL: process.env.NIMI_API_DEV_BASE_URL,
     APP_ENV: process.env.APP_ENV,
-    POAP_API_KEY: process.env.POAP_API_KEY,
+    NEXT_POAP_API_KEY: process.env.NEXT_POAP_API_KEY,
```

Comment:
Should be `NEXT_PULIBC_*` prefix so this variable is also available in the front-end

Reply:
also wondering why did you add this poap api key here?
![Screenshot 2023-02-06 at 16 45 41](https://user-images.githubusercontent.com/33226956/217017414-7339dd34-1c2a-4422-8a28-55e1f25c8b66.png)

Reply:
pushed this as well this was causing poaps not to show...